Document CORS tip

choldgraf opened the pull request:
This adapts and documents some useful CORS tips that @rowanc1 posted in the following issue:

- https://github.com/the-turing-way/the-turing-way/pull/4156

rowanc1 commented on docs/website-metadata.md:
```diff
@@ -104,3 +104,27 @@ references
+
+If your MyST site is public and does not require authentication, allowing all origins does not pose a security risk.
+
+_Note, this was adapted from [this GitHub issue from @rowanc1](https://github.com/the-turing-way/the-turing-way/pull/4156)_.
```
Can remove this! Thanks for bringing it in.

Or maybe change it to a link as a "for example implementation"?

choldgraf replied:
oh good idea - will do that

choldgraf commented:
I updated the reference to the PR so that it is used as an example, and also added a link in Netlify. Thanks for the suggestion! I'll merge since this already has a +1
